popup.html

```diff
@@ -1,17 +1,18 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>WakaTime</title>
-
-    <link href="public/css/app.css" rel="stylesheet">
-</head>
-<body>
-
-<div id="wakatime"></div>
-
-<script src="public/js/app.js"></script>
-</body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>WakaTime</title>
+
+    <link href="public/css/app.css" rel="stylesheet">
+    <script src="public/js/browser-polyfill.min.js"></script>

```

Comment:
I had thought that that is why you would use Load/DomContentLoaded. Looks like adding it to the end can be better as it won't block page render due to a download. It's probably best to block to ensure the polyfill is in place. I'm not sure how large it is at the moment, but I don't expect it to be too big.